specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -0,0 +1,660 @@
+        "operationId": "DataRegistries_GetContent",
+        "x-ms-examples": {
+          "Download previously registered GeoJSON content": {
+            "$ref": "./examples/DataRegistries_GetContent.json"
+          }
+        },
+        "produces": [
+          "application/geo+json",
+          "application/octet-stream"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/UserDataId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get content request completed successfully. The response body will contain the content for the passed in `udid`.",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "description": "The content-type for the Download API response."
+              }
+            }
+          },
+          "404": {
+            "description": "Resource not ready or not found."
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../Common/preview/1.0/common.json#/definitions/MapsErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataRegistry": {
+      "description": "The data registry in Azure Maps.",
+      "type": "object",
+      "properties": {
+        "udid": {
+          "description": "The user data id for the data registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The description to be given to the data registry. The max length is 1024 characters.",
+          "type": "string"
+        },
+        "dataFormat": {
+          "$ref": "#/definitions/DataFormat"
+        },
+        "connection": {
+          "$ref": "#/definitions/Connection"
+        },
+        "downloadURL": {
+          "description": "The download URL of the underlying data. It may not be available if the status of the data registry is `Running` or `Failed`. Note that `api-version` is not included.",
```

Comment:
per Will's implementation, the api-version is included. only subscription-key is not included.

Reply:
Fixed